src/examples/sqrt-task-network/worker/README.md

```diff
@@ -0,0 +1,71 @@
+# Sqrt Task Network Worker
+
+This is a worker implementation for the Sqrt Task Network example. The worker listens for new tasks, computes square roots, and submits the results back to the blockchain.
+
+## Prerequisites
+
+- Docker
+- Python 3.11+
+- Access to an Holesky Ethereum node
+- Private keys for validator and operator accounts
```

Comment:
How to get operator account (link to docs); and what is private key for validator here